src/transformers/modeling_tf_bert.py

```diff
@@ -23,7 +23,13 @@
 
 from .configuration_bert import BertConfig
 from .file_utils import MULTIPLE_CHOICE_DUMMY_INPUTS, add_start_docstrings, add_start_docstrings_to_callable
-from .modeling_tf_utils import TFPreTrainedModel, get_initializer, keras_serializable, shape_list
+from .modeling_tf_utils import (
```

Comment:
@Bharat123rox here you can see how `output_attentions` should be applied to a TF model.

You have to do the following:
1) Make sure that `output_attentions` is passed to all forward functions as a list parameter
2) cast all `output_attentions` booleans using the function `cast_bool_to_primitive`
3) extract `output_attentions` from a dict or list if necessary